scripts/spack/configs/linux_ubuntu_24/spack.yaml

```diff
@@ -1,36 +1,21 @@
-# Copyright (c) 2017-2025, Lawrence Livermore National Security, LLC and
+# Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
 # other Tribol Project Developers. See the top-level LICENSE file for details.
 #
 # SPDX-License-Identifier: (MIT)
 
 spack:
   compilers::
   - compiler:
-      spec: gcc@11.4.0
+      spec: clang@16.0.6
```

Comment:
Can you clarify this change. Are we no longer defaulting to gcc and instead using clang for enzyme support? Does this sort of thing need to be updated elsewhere and not just for linux_ubuntu_22?

Reply:
The ubuntu 24 host config is so I can do code development on my local machine, so the options mirror what are on my machine.  It is provided in the repo for the benefit of other users who want to build TPLs on this platform, but is not guaranteed to work out of the box.